crates/api_common/src/custom_emoji.rs

```diff
@@ -46,3 +47,21 @@ pub struct DeleteCustomEmoji {
 pub struct CustomEmojiResponse {
   pub custom_emoji: CustomEmojiView,
 }
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+#[cfg_attr(feature = "full", derive(TS))]
+#[cfg_attr(feature = "full", ts(export))]
+/// A response for custom emojis.
+pub struct ListCustomEmojisResponse {
+  pub custom_emojis: Vec<CustomEmojiView>,
+}
+
+#[skip_serializing_none]
+#[derive(Debug, Serialize, Deserialize, Clone, Copy, Default, PartialEq, Eq, Hash)]
+#[cfg_attr(feature = "full", derive(TS))]
+#[cfg_attr(feature = "full", ts(export))]
+/// Fetches a list of registration applications.
+pub struct ListCustomEmojis {
+  pub page: Option<i64>,
```

Comment:
@makotech222 look into your emoji library, because these may need a search term, optional category, etc.

Reply:
I've added a category search parameter. This column is also already indexed